pyquil/operator_estimation.py

```diff
@@ -746,10 +762,24 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
         to True is much faster but there is a ~1% error per qubit in the reset operation.
         Thermal noise from "traditional" reset is not routinely characterized but is of the same
         order.
+    :param symmetrize: Whether to symmetrize the readout errors, i.e. set p(0|1) = p(1|0). For
```

Comment:
there has been requests in the past to be explicit about calling this *readout* symmetrization, as there are many things you could conceivably symmetrize

Reply:
for forwards compatibility with additional readout symmetrization designs, consider making this parameter a string (ie `"exhaustive"` or `"none"`)

Reply:
Changed both `readout_symmetrize` as well as `calibrate_readout` to str arguments for forward compatibility.